app/services/form_to_resource_service.rb

```diff
@@ -113,5 +114,20 @@ def new_contributor(given_name, family_name, orcid, type, sequence)
         return if family_name.blank? && given_name.blank? && orcid.blank?
         PDCMetadata::Creator.new_contributor(given_name, family_name, orcid, type, sequence)
       end
+
+      # Funders:
+
+      def process_funders(params, resource)
+        # (New pattern: Use rails param name conventions rather than numbering fields.)
```

Comment:
I'm unsure what the params are going to look like here.  That does not indicate a code change as long as it is show in the tests.